templates/_macros/hero.jinja

```diff
@@ -0,0 +1,127 @@
+# Params
+# title_text: Hero title text (required)
+# subtitle_text: Hero subtitle text
+# layout: layout of hero section. Options are '50/50', '75/25', '25/75', 'fallback'
+# is_split_on_tablet: whether the layout is split on tablet. Applies "-on-medium" to row class in 50/50 and 25/75 layouts. If false, the layout is stacked on tablet.
+# is_split_on_desktop: whether the layout is split on desktop. Applies "-on-large" to row class in 50/50 and 25/75 layouts. If false, the layout is stacked on desktop, provided is_split_on_tablet is true.
+# is_image_full_cover: whether the image should span full width of the hero. Only supported on 50/50 layout
+# cta_primary_url: primary CTA button URL
+# cta_secondary_url: secondary CTA button URL
+# Slots
+# description: paragraph-style content below the title and subtitle
+# cta_primary: contents of the primary CTA button
+# cta_secondary: contents of the secondary CTA button
+# image: slot for image content (required)
+
+{% macro hero(
+  title_text='',
+  subtitle_text='',
+  layout="fallback",
+  is_split_on_tablet = false,
+  is_split_on_desktop = true,
+  is_image_full_cover = false,
```

Comment:
"Full cover" sounds like it turns on the "cover" feature on the image itself, while it's an option for layout.

I'd suggest `is_image_full_width` or `has_full_width_image`